locksmith/src/operations/userOperations.ts

```diff
@@ -0,0 +1,47 @@
+const ethJsUtil = require('ethereumjs-util')
+const models = require('../models')
+const { User, UserReference } = models
+import * as types from '../types'
+
+namespace UserOperations {
+  export async function createUser(
+    input: types.UserCreationInput
+  ): Promise<Boolean> {
+    let userReference = await UserReference.create(
+      {
+        emailAddress: input.emailAddress.toLowerCase(),
```

Comment:
I don't remember seeing a test where we check with an UPPERCASE email address and we still return the private key corresponding to the lowercase one? Could you add this please?